extensions/sql-migration/src/dialog/migrationStatus/migrationStatusDialog.ts

```diff
@@ -473,23 +508,55 @@ export class MigrationStatusDialog {
 			.component();
 
 		if (count > 0) {
-			control.addItems([
-				// migration warning / error image
-				this._view.modelBuilder.image().withProps({
+			const migrationWarningImage = this._view.modelBuilder.image()
+				.withProps({
 					iconPath: this._statusInfoMap(status),
 					iconHeight: statusImageSize,
 					iconWidth: statusImageSize,
 					height: statusImageSize,
 					width: statusImageSize,
 					CSSStyles: imageCellStyles
-				}).component(),
-				// migration warning / error counts
-				this._view.modelBuilder.text().withProps({
-					value: loc.STATUS_WARNING_COUNT(status, count),
+				}).component();
+
+			const migrationWarningCount = this._view.modelBuilder.hyperlink()
+				.withProps({
+					label: loc.STATUS_WARNING_COUNT(status, count) ?? '',
+					url: '',
```

Comment:
Make sure you verify proper accessibility support (keyboard navigation + screen reader announce), I don't think we've done a callout dialog from a hyperlink before so there might be issues.

Reply:
Yes, it is accessible to open and close using keyboard navigation